deploy/secrets-store.csi.x-k8s.io_secretproviderclasses.yaml

```diff
@@ -61,6 +61,11 @@ spec:
                           type: string
                       type: object
                     type: array
+                  labels:
```

Comment:
Try a rebase with master and then generating the manifests using `make manifests`. With #258 we have added `manifest_staging` dir to host the chart/deploy changes. These changes will be tagged with the release. Only manifests in `manifest_staging` need to be changed as part of the PR